BUILD.bazel

```diff
@@ -188,6 +188,22 @@ cc_library(
     ],
 )
 
+###############################################################################
+# For testing
+###############################################################################
+
+cc_library(
+	name = "private_test_headers",
+	hdrs = glob(["test/*.h"]),
+	copts = COPTS,
+	strip_include_prefix = "test",
+	visibility = ["//visibility:public"],
```

Comment:
This library only contains the `test/assert_compare.h` header file that is used in serveral test under regression folder.

I tried using `visibility:private` but I can't add this target as `dep` to a `cc_test` statement because there is no visibility.

Reply:
Thanks! Done at [ffa0582](https://github.com/maliput/maliput_malidrive/pull/246/commits/ffa05829ff09b06d4f82e47650701e4f4d2e7e92)